schema.json

```diff
@@ -232,18 +231,54 @@
                 "uri": {
                     "type": "string"
                 },
-                "policy_document": {
+                "bastion_ip": {
+                    "type": "string"
+                },
+                "bastion_private_key": {
                     "type": "string"
                 }
             },
             "required": [
                 "user",
                 "host",
                 "port",
-                "name",
+                "database_name",
                 "cluster_arn",
                 "uri"
             ]
+        },
+        "nuage:aws:Bastion": {
```

Comment:
Could you please add a description with a code example that showcases using the `vpc.private_subnet_ids` for the DB and the `vpc.public_subnet_ids[0]` for the bastion ?